hack/errata.py

```diff
@@ -55,45 +55,56 @@ def run(poll_period=datetime.timedelta(seconds=3600),
         githubtoken=None,
         **kwargs):
     next_time = datetime.datetime.now()
+    if excluded_cache is None:
```

Comment:
I guess it would be easier to set `excluded_cache={}` in function declaration instead.